### PR TITLE
fix(laravel sanctum): set `referer` header on mount

### LINKED ISSUE #653: keep redirecting me to login after refresh
after successfully login and redirect to /home (with middleware:auth set) after refresh i cant acces home route anymore it keeps redirect me to login.i use laravel sanctum

### PR DESCRIPTION
**Pull request by JoaoPedroAS51**
Seems that Laravel Sanctum requires `referer` header in SSR request to authorize the request. This fix will set `referer` header on mount. 
This also set token `maxAge` to `false` as we are not using token.

Fixes #653

### CODE REVIEW

File: src/schemes/cookie.ts [hunk 1]
```diff
@@ -16,8 +18,20 @@ const DEFAULTS = {
...
+
   constructor ($auth, options) {
     super($auth, options, DEFAULTS)
+
+    this.req = $auth.ctx.req
+  }
+
+  mounted () {
+    if (process.server) {
+      this.$auth.ctx.$axios.setHeader('referer', requrl(this.req))
```
Comment (pi0) [inline src/schemes/cookie.ts:31]: Can we directly use `this.$authx.ctx.req` here? Also might be a good idea linking issue with a comment on why we would need to set referrer.

Comment (pi0) [reply src/schemes/cookie.ts:31]: One more question: Is refer check only by hostname or full path?

Comment (JoaoPedroAS51) [reply src/schemes/cookie.ts:31]: Ok

Comment (JoaoPedroAS51) [reply src/schemes/cookie.ts:31]: I think it's only hostname. At least it woked for me and @radudiaconu0

**Pull request merged**